src/jit-diff/diff.cs

```diff
@@ -390,6 +390,39 @@ public static int DiffCommand(Config config)
                 }
             }
 
+            private static int CompareFrameworkAssemblies(AssemblyInfo assemblyInfo1, AssemblyInfo assemblyInfo2)
+            {
+                string assembly1 = assemblyInfo1.Path;
+
+                if (assembly1 == s_CoreLibAssembly)
+                {
+                    return -1;
+                }
+
+                string assembly2 = assemblyInfo2.Path;
+
+                if (assembly2 == s_CoreLibAssembly)
+                {
+                    return 1;
+                }
+
+                long length1 = new FileInfo(assembly1).Length;
+                long length2 = new FileInfo(assembly2).Length;
```

Comment:
Calling this in the sort compare function will create a large number of unused FileInfo objects.

Reply:
They will be used, just very briefly, and then collected. In our scenario the sort happens instantaneously, I don't think this is worth optimizing.